pippo-core/src/main/java/ro/pippo/core/Request.java

```diff
@@ -391,6 +392,19 @@ public String getContentType() {
         return contentType;
     }
 
+    /**
```

Comment:
It's enough as javadoc something like: "It's wrapper over {@link HttpServletRequest#getLocale}."

Reply:
Done!